src/tests/ftest/rebuild/widely_striped.yaml

```diff
@@ -2,6 +2,9 @@ hosts:
   test_servers: 7
   test_clients: 1
 timeout: 1200
+agent_config:
+  cache_expiration: 1
```

Comment:
Out of scope for this PR, but does/should the config accept `1m` to be clear?

Reply:
Not sure. Thinking probably not, since this is a bit of a "pass through" to what the daos_agent.yml configuration is, and the documentation for that config parameter (utils/config/daos_agent.yml file example) indicates the integer value is specified in minutes. So my presumption is that a value with units is not expected at the control plane / Go code level. If we were to do this in the ftest configuration and code, then it seems it would add clarity to the human reader of a given .yaml file but would introduce some ftest python logic to manage that.

I wonder if it would be useful to have a sample ftest .yaml file that presents various daos-level configuration options that a test might need to configure, presented with comments for documentation, similar to what is done for the product .yml files?